src/lib/Commands/TestCommand/index.js

```diff
@@ -10,54 +33,56 @@ const fs = require('fs');
 const c = require('colors');
 const path = require('path');
 const glob = require('glob');
+const Builder = require('Builder');
 const Errors = require('./Errors');
 const Session = require('./Session');
-const Bundler = require('../../Bundler');
 const dateformat = require('dateformat');
 const LogParser = require('./LogParser');
 const Watchdog = require('../../Watchdog');
 const randomstring = require('randomstring');
 const sprintf = require('sprintf-js').sprintf;
 const BuildAPIClient = require('imp-build-api-v4');
-const CodeProcessor = require('../../CodeProcessor');
 const AbstractCommand = require('../AbstractCommand');
 const promiseWhile = require('../../utils/promiseWhile');
 //</editor-fold>
 
-/**
- * Delay before testing start.
- * Prevents log sessions mixing, allows
- * service messages to be before tests output.
- * [s]
- */
+
+// Delay before testing start.
+// Prevents log sessions mixing, allows
+// service messages to be before tests output.
+// [s]
+
 const DEFAULT_STARTUP_DELAY = 2;
 
-/**
- * Timeout before session startup
- */
+// Timeout before session startup
+
 const DEFAULT_STARTUP_TIMEOUT = 60;
 
-/**
- * Allow extra time on top of .imptest.timeout before
- * treating test as timed out on a tool siode.
- */
+// Allow extra time on top of .imptest.timeout before
+// treating test as timed out on a tool siode.
+
 const DEFAULT_EXTRA_TEST_MESSAGE_TIMEOUT = 5;
 
-/**
- * Name for BuildAPI key env var
- */
+// Name for BuildAPI key env var
+
 const BUILD_API_KEY_ENV_VAR = 'IMP_BUILD_API_KEY';
 
-/**
- * Test command
- */
+// Unsupported in Builder symbols
+
+const UNSUPPORTED_SYMBOLS_REGEXP = /[^_A-Za-z0-9]/g;
```

Comment:
I still believe you don't need to replace any symbols in the variable names/values. Are there any use cases when it's needed?